### DISCUSSION TITLE
How to convert the coordinate direction of texture data measured by xrd in mtex

### DISCUSSION THREAD

**Discussion by ddyyasfh** [mtex-toolbox/mtex#1769] [Ask Anything] [answered]
Hi everyone
  How to convert the coordinate direction of texture data measured by XRD in mtex?

Thank you,
Dy.

**Comment by kilir** [comment, marked as answer]
Hi,
you can apply a rotation, see here: https://mtex-toolbox.github.io/RotationDefinition.html
Cheers,
Rüdiger
